README.md

```diff
@@ -178,13 +178,13 @@ Node module for integration with Spree API V2
 
 ## Endpoints
 
-Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides2.spreecommerce.org/api/v2)
+Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides.spreecommerce.org/api/v2/storefront/)
 
 ## Authentication
 
 ### `getToken`
 
-Method `getToken` used for creates a Bearer token required to authorize calls API calls. [Read more](https://guides2.spreecommerce.org/api/v2/authentication)
+Method `getToken` used for creates a Bearer token required to authorize calls API calls. [Read more](https://guides.spreecommerce.org/api/v2/storefront//authentication)
```

Comment:
wrong link to `authentication`, should be `https://guides.spreecommerce.org/api/v2/storefront/#section/Authentication`